helpers/config-loader.js

```diff
@@ -1,4 +1,9 @@
-module.exports = function(file, plugins, config) {
+'use strict';
+module.exports = function (file, plugins, config, failOnError) {
+  if (typeof failOnError === 'undefined') {
+    failOnError = true;
+  }
```

Comment:
You can use ES6 default parameter value instead this.

https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Functions/Default_parameters